lib/site_prism/page.rb

```diff
@@ -10,7 +10,10 @@ class Page
     extend ElementContainer
 
     load_validation do
-      [displayed?, "Expected #{current_url} to match #{url_matcher} but it did not."]
+      [
```

Comment:
This change made Page class longer than 100 lines. This is the reason rubocop is failing